site/content/recipes/find_a_reviewer.md

```diff
@@ -0,0 +1,184 @@
+enough to have a Research Software Engineering (RSE) group at your institution,
+then any member of that team would make a great reviewer for your code.
+These groups have many different ways of operating, so you will need to contact
+them to find out how to set up a code review. Forewarning: it is likely the
+time of the RSE will need to be covered under someone's budget, so if you were
+hesitant to have the conversation about code review in your research group,
+that problem will probably resurface here.
+
+Another great place to search within your institution or region are language or
+library-specific community groups (e.g. a Python or R User Group). This can
+help ensure the reviewer has experience with the same or similar tools you use.
+Before [making your advertisement](#contact), consider asking one of the
+community managers or someone you know in the community whether code review is
+something that has been done before or others might be interested in. Just as
+in the case with colleagues, you will be relying on people to volunteer their
+time, and as you are probably all to familiar with, researchers have that in
+very short supply. The community simply might not have the bandwidth to take on
+code review.
+
+It might be necessary to venture outside of your institution or immediate
+geographic region to find a reviewer, and it is an increasingly viable option
+in the context of remote working and many people getting used to asynchronous
+communication. If you are part of a research organisation, community, or
+working group that spans institutions and countries, this could be a viable
+avenue to search for a reviewer. Your connections in these areas might not
+themselves be suitable, but you can ask if they know anyone who might be.
+Open source or open science communities could similarly be leveraged,
+especially if you intend to make your software open source. If you have
+extended an open source library or even just made some convenient scripts to
+work with it, consider opening a merge/pull request to get the maintainers to
+have a look. A good example of this for more mature software is [rOpenSci's
+peer review initiative](https://bssw.io/blog_posts/a-community-of-practice-around-peer-review-for-long-term-research-software-sustainability).
+
+If none of the above options bear any fruit, then as a last resort you could
+make a post on the [Code Review Stack
+Exchange](https://codereview.stackexchange.com/). See the ["What questions can
+I ask about here?"](https://codereview.stackexchange.com/help/on-topic) page to
+determine whether it could suit your needs.
+
+## Step 2: Criteria for Screening {#criteria}
+
+Once you have identified a potential reviewer and before you contact them, you
+should consider whether they will satisfy some minimum thresholds to be a
+suitable reviewer. These criteria equally apply when deciding whether to
+advertise to a larger group.
```

Comment:
I feel this reads a bit too cold and analytical as if reviewers could just be sorted by skill. We could soften it a bit by making it explicit that it's about their suitability to the code being reviewed. Also, it would read better to list the three criteria at the start before going in detail about each one. Maybe something like "Before contacting a potential reviewer, consider whether they are a good fit to the code being reviewed. Ideally, a reviewer will be experience in the programming language, in the external libraries or frameworks used, and in the problem domain. This also applies when advertising to a larger group."

Reply:
> Before contacting a potential reviewer, consider whether they are a good fit to the code being reviewed

I like that!

> Ideally, a reviewer will be experience in the programming language, in the external libraries or frameworks used, and in the problem domain.

I guess this depends on the type of feedback that is looked for. In my experience having my code reviewed by somebody who' s less familiar with the language is a very good assessment of readability.

Reply:
This section has been "softened". Thanks for the suggestions David (can't seem to @ you).